### ISSUE TITLE
Report Generation

### ISSUE THREAD

**Issue by kamal2222ahmed** [s-chathuranga-j/accessibilityAutomation-selenium-java-axeCore4plus#1]
I have the report in json and txt format, 
what do I do next? how do I create a report preferably in HTML format. I tried to change the URL but still not get an updated json or txt file. in the directory: src/test/java/reports 
accessibilityReport.json	
accessibilityReport.txt
These 2 files did not get updated 

I used:

```
public class AmazonOfficial extends FrameworkBase
{
    AccessibilityBase a11y;

    @Test
    public void testAmazon() throws IOException {
        webDriver().get("https://www.cnn.com");
        a11y.checkAccessibilityViolations();
    }
}
```
